src/core.js

```diff
@@ -516,16 +511,18 @@ function extractStacktrace( e, offset ) {
 }
 
 function sourceFromStacktrace( offset ) {
-	var e = new Error();
-	if ( !e.stack ) {
+	var error = new Error();
+
+	// Ref https://github.com/jquery/qunit/issues/636
```

Comment:
Just noticed the detail in the PR description. Let's make that into another support comment.